packages/nouns-contracts/contracts/governance/NounsDAOProposals.sol

```diff
@@ -731,6 +733,9 @@ library NounsDAOProposals {
         for (uint256 pid = firstProposalId; pid <= lastProposalId; ++pid) {
             proposal = ds._proposals[pid];
 
+            if (excludeCanceled && proposal.canceled) continue;
```

Comment:
I don't think we have a test that runs through what happens when this exclude param is false?

Reply:
added a test: d003fe857d65dc76c8a60ef6c0d57ad4aae896f3